python/py-genshi/Portfile

```diff
@@ -1,17 +1,18 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           github 1.0
 PortGroup           python 1.0
 
 name                py-genshi
-version             0.7
+github.setup        edgewall genshi 0.7.7
 revision            0
 categories-append   textproc www
 platforms           {darwin any}
 supported_archs     noarch
 license             BSD
 
-python.versions     27 35 36
+python.versions     27 39 310 311
 
```

Comment:
please add `python.pep517` yes, as the project provides a `pyproject.toml` file

Reply:
ah, sorry that doesn't work for the PY27 subport so that would need a special case.

Reply:
Ok, so how to do that?

In fact, it installed for me as `py311-genshi` without setting `python.pep517`. I thought it is a default anyway.

Reply:
It is the default for py312 subports, not for the others

Reply:
Just exclude py27 one?

Reply:
```
if {${name} ne ${subport}} {
    if {${python.version} > 27} {
        python.pep517 yes
    }
}
```

Reply:
Added.